diff --git a/deps/Makefile b/deps/Makefile
index 5f7bddb..948929b 100644
--- a/deps/Makefile
+++ b/deps/Makefile
@@ -673,6 +673,11 @@ $(eval $(call LLVM_PATCH,win64-int128.llvm-3.3))
 else ifeq ($(LLVM_VER),3.6.0)
 $(eval $(call LLVM_PATCH,zerosign-llvm-3.6.0))
 $(eval $(call LLVM_PATCH,win64-allocas-llvm-3.6.0))
+else ifeq ($(LLVM_VER),svn)
+$(LLVM_SRC_DIR)/lldb-python.patch-applied: $(LLVM_SRC_DIR)/configure | $(SRCDIR)/lldb-python.patch
+	cd $(LLVM_SRC_DIR)/tools/lldb && patch -p0 < $(SRCDIR)/lldb-python.patch
+	echo 1 > $@
+LLVM_PATCH_LIST += $(LLVM_SRC_DIR)/lldb-python.patch-applied
 endif # LLVM_VER
 
 LLVM_BUILDDIR_withtype := $(LLVM_BUILD_DIR)/build_$(LLVM_BUILDTYPE)
