Index: include/lldb/API/SBDefines.h
===================================================================
--- include/lldb/API/SBDefines.h
+++ include/lldb/API/SBDefines.h
@@ -86,9 +86,7 @@
 class LLDB_API SBTypeNameSpecifier;
 class LLDB_API SBTypeSummary;
 class LLDB_API SBTypeSummaryOptions;
-#ifndef LLDB_DISABLE_PYTHON
 class LLDB_API SBTypeSynthetic;
-#endif
 class LLDB_API SBTypeList;
 class LLDB_API SBValue;
 class LLDB_API SBValueList;
Index: include/lldb/API/SBTypeSummary.h
===================================================================
--- include/lldb/API/SBTypeSummary.h
+++ include/lldb/API/SBTypeSummary.h
@@ -12,8 +12,6 @@
 
 #include "lldb/API/SBDefines.h"
 
-#ifndef LLDB_DISABLE_PYTHON
-
 namespace lldb {
     class LLDB_API SBTypeSummaryOptions
     {
@@ -163,6 +161,4 @@
     
 } // namespace lldb
 
-#endif // LLDB_DISABLE_PYTHON
-
 #endif // LLDB_SBTypeSummary_h_
Index: include/lldb/API/SBValue.h
===================================================================
--- include/lldb/API/SBValue.h
+++ include/lldb/API/SBValue.h
@@ -139,10 +139,8 @@
     lldb::SBTypeFormat
     GetTypeFormat ();
     
-#ifndef LLDB_DISABLE_PYTHON
     lldb::SBTypeSummary
     GetTypeSummary ();
-#endif
 
     lldb::SBTypeFilter
     GetTypeFilter ();
Index: include/lldb/DataFormatters/TypeSummary.h
===================================================================
--- include/lldb/DataFormatters/TypeSummary.h
+++ include/lldb/DataFormatters/TypeSummary.h
@@ -541,8 +541,6 @@
         DISALLOW_COPY_AND_ASSIGN(CXXFunctionSummaryFormat);
     };
     
-#ifndef LLDB_DISABLE_PYTHON
-    
     // Python-based summaries, running script code to show data
     struct ScriptSummaryFormat : public TypeSummaryImpl
     {
@@ -614,7 +612,6 @@
     private:
         DISALLOW_COPY_AND_ASSIGN(ScriptSummaryFormat);
     };
-#endif
 } // namespace lldb_private
 
 #endif // lldb_TypeSummary_h_
Index: include/lldb/lldb-forward.h
===================================================================
--- include/lldb/lldb-forward.h
+++ include/lldb/lldb-forward.h
@@ -186,9 +186,7 @@
 class   Scalar;
 class   ScriptInterpreter;
 class   ScriptInterpreterLocker;
-#ifndef LLDB_DISABLE_PYTHON
 struct  ScriptSummaryFormat;
-#endif
 class   SearchFilter;
 class   Section;
 class   SectionImpl;
@@ -384,9 +382,7 @@
     typedef std::shared_ptr<lldb_private::Queue> QueueSP;
     typedef std::weak_ptr<lldb_private::Queue> QueueWP;
     typedef std::shared_ptr<lldb_private::QueueItem> QueueItemSP;
-#ifndef LLDB_DISABLE_PYTHON
     typedef std::shared_ptr<lldb_private::ScriptSummaryFormat> ScriptSummaryFormatSP;
-#endif // #ifndef LLDB_DISABLE_PYTHON
     typedef std::shared_ptr<lldb_private::ScriptInterpreter> ScriptInterpreterSP;
     typedef std::unique_ptr<lldb_private::ScriptInterpreter> ScriptInterpreterUP;
     typedef std::shared_ptr<lldb_private::Section> SectionSP;
Index: source/API/SBTypeSummary.cpp
===================================================================
--- source/API/SBTypeSummary.cpp
+++ source/API/SBTypeSummary.cpp
@@ -15,8 +15,6 @@
 using namespace lldb;
 using namespace lldb_private;
 
-#ifndef LLDB_DISABLE_PYTHON
-
 SBTypeSummaryOptions::SBTypeSummaryOptions()
 {
     m_opaque_ap.reset(new TypeSummaryOptions());
@@ -434,5 +432,3 @@
     
     return true;
 }
-
-#endif // LLDB_DISABLE_PYTHON
Index: source/API/SBValue.cpp
===================================================================
--- source/API/SBValue.cpp
+++ source/API/SBValue.cpp
@@ -632,7 +632,6 @@
     return result;
 }
 
-#ifndef LLDB_DISABLE_PYTHON
 const char *
 SBValue::GetSummary ()
 {
@@ -681,7 +680,6 @@
     }
     return cstr;
 }
-#endif // LLDB_DISABLE_PYTHON
 
 const char *
 SBValue::GetLocation ()
@@ -753,7 +751,6 @@
     return format;
 }
 
-#ifndef LLDB_DISABLE_PYTHON
 lldb::SBTypeSummary
 SBValue::GetTypeSummary ()
 {
@@ -771,7 +768,6 @@
     }
     return summary;
 }
-#endif // LLDB_DISABLE_PYTHON
 
 lldb::SBTypeFilter
 SBValue::GetTypeFilter ()
Index: source/DataFormatters/TypeSummary.cpp
===================================================================
--- source/DataFormatters/TypeSummary.cpp
+++ source/DataFormatters/TypeSummary.cpp
@@ -205,9 +205,6 @@
     return sstr.GetString();
 }
 
-#ifndef LLDB_DISABLE_PYTHON
-
-
 ScriptSummaryFormat::ScriptSummaryFormat (const TypeSummaryImpl::Flags& flags,
                                           const char * function_name,
                                           const char * python_script) :
@@ -275,5 +272,3 @@
     return sstr.GetString();
     
 }
-
-#endif // #ifndef LLDB_DISABLE_PYTHON
